swaymsg/swaymsg.1.scd

```diff
@@ -82,3 +82,7 @@ _swaymsg_ [options...] [message]
 	Subscribe to a list of event types. The argument for this type should be
 	provided in the form of a valid JSON array. If any of the types are invalid
 	or if an valid JSON array is not provided, this will result in an failure.
+
+# SEE ALSO
+
+*sway*(5) *swat-bar*(5) *sway-input*(5) *sway-output*(5) *sway-ipc*(7)
```

Comment:
```suggestion
*sway*(5) *sway-bar*(5) *sway-input*(5) *sway-output*(5) *sway-ipc*(7)
```

Reply:
Thanks! Fixed, fixed up, and force pushed